include/desktop/output.h

```diff
@@ -24,6 +24,7 @@ struct kiwmi_output {
     struct wlr_box usable_area;
 
     int damaged;
+    int enable_changed;
```

Comment:
Should be a bool if i’m not mistaken.

(`damaged` holds 0, 1, or 2 depending on the number of frames to redraw, so it needs to be (some sort of) int)

Reply:
I'm not too happy about putting it there either, but the alternative is creating some complex data structure in the `output_manager_configure` function to track which outputs have been changed.  This seemed like the most keep-it-simple solution.

I have made it a bool now though.

Reply:
I completely get that. Can you add a comment about its usage, optionally including the arguments for this state?